src/main/python/systemds/matrix/data_gen.py

```diff
@@ -19,7 +19,7 @@
 #
 # -------------------------------------------------------------
 
-__all__ = ['full', 'seq', 'rand']
+__all__ = ['full', 'seq', 'rand', 'rev', 'order', 't', 'cholesky']
```

Comment:
This file contains methods for data generation, not for operations.
I think it would be nice if you could move the `rev` `order` `t` and `cholesky` to the matrix.py object. thereby removing the need and potential error associated with calling these functions with an SystemDSContext.